packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -234,6 +264,9 @@ class InputDateTimePicker extends React.Component {
 								useTime={this.props.useTime}
 								useSeconds={this.props.useSeconds}
 								useUTC={this.props.useUTC}
+								onFinish={this.onFinish}
+								isValid={!this.state.errorMessage}
+								formMode={this.props.formMode}
```

Comment:
I think you can avoid passing `formMode` and `onSubmit` (onFinish).
It can be the responsibility of `InputDatePicker` to display the form mode.
- wrap the date picker with a form,
- insert the submit button in footer
- insert the error message in footer
- manage a proper form onSubmit attribute so it handles the time inputs enter naturally

Reply:
> insert the error message in footer
was thinking of doing that in another PR